test/extended/testdata/bindata.go

```diff
@@ -11597,7 +11597,8 @@ var _examplesDbTemplatesMariadbEphemeralTemplateJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Username: ${MYSQL_USER}\n       Password: ${MYSQL_PASSWORD}\n  Database Name: ${MYSQL_DATABASE}\n Connection URL: mysql://${DATABASE_SERVICE_NAME}:3306/\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/blob/master/10.1/README.md.",
   "labels": {
-    "template": "mariadb-persistent-template"
+    "template": "mariadb-persistent-template",
+    "app": "mariadb-persistent-template"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.